model/src/main/java/com/ontimize/mbx/model/core/service/ToolService.java

```diff
@@ -0,0 +1,32 @@
+package com.ontimize.mbx.model.core.service;
+
+import org.springframework.stereotype.Service;
+
+import com.ontimize.db.SQLStatementBuilder.BasicExpression;
+import com.ontimize.db.SQLStatementBuilder.BasicField;
+import com.ontimize.db.SQLStatementBuilder.BasicOperator;
+import com.ontimize.mbx.api.core.service.ITool;
+
+@Service("ToolService")
+public class ToolService implements ITool{
+
+	@Override
+	public BasicExpression searchObjectByLikeName(String attrName, String object_name) {
+
+		BasicField field = new BasicField(attrName);
+		String[] data = object_name.trim().split("\\s+");
+		BasicExpression create = null;
+		BasicExpression send = null; 
+		BasicExpression acumuled = null;
+
+		for (String d : data) {
+			if(send == null) {
+				send = new BasicExpression(field, BasicOperator.LIKE_OP, "%"+ d + "%");
+			} else {	
+				create = new BasicExpression(field, BasicOperator.LIKE_OP, "%"+ d + "%");
```

Comment:
Hola. Si solo usas la variable de create dentro del for deberíais crearla dentro del for. Luego si no usáis la de acumuled, pues no la creéis.